README.md

```diff
@@ -22,7 +22,7 @@ Table of Contents
    * [Testing](#testing)
    * [BaaS](#baas)
    * [CDN and Protection](#cdn-and-protection)
-   * [CI / CD](#cicd)
+   * [CI / CD](#ci-cd)
```

Comment:
this wont work because github and the website has different rules, only fix is to change to `CI and CD`